gcc/common/config/aarch64/aarch64-common.cc

```diff
@@ -426,6 +426,8 @@ aarch64_rewrite_mcpu (int argc, const char **argv)
   return aarch64_rewrite_selected_cpu (argv[argc - 1]);
 }
 
+#if TARGET_64BIT_MS_ABI
+
 /* Implement TARGET_EXCEPT_UNWIND_INFO.  */
 
 static enum unwind_info_type
```

Comment:
I am curious why unwind_info_type has not been defined earlier for aarch64

Reply:
This PR looks like a temporary solution for me to unblock validation for linux-gnu